src/extensions/default/Brackets-InteractiveLinter/linterManager.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * Interactive Linter Copyright (c) 2015 Miguel Castillo.
+ *
+ * Licensed under MIT
+ */
+
+
+define(function (require /*, exports, module*/) {
+    "use strict";
+
+    var _                  = brackets.getModule("thirdparty/lodash"),
+        LanguageManager    = brackets.getModule("language/LanguageManager"),
+        PreferencesManager = brackets.getModule("preferences/PreferencesManager"),
+        preferences        = PreferencesManager.getExtensionPrefs("interactive-linter"),
```

Comment:
We might need to fix some code here to have it `require` vs. loading `brackets.json`, which I think this does.